alpine-apache/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM smebberson/alpine-base:1.1.0
```

Comment:
This has since been upgraded to `alpine-base:1.2.0`. Would you mind updating accordingly?